hackathon-topics.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "title": "Example Topic",
+        "description": "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.",
+        "host": "Some Guy",
+        "homepage": "https://github.com/cko/nixcon2017",
+        "skillLevel": 1,
+        "skillLevelTo": 3
+    }
```

Comment:
Please add duration and day, cause not everyone might want to discuss a topic for two days and people might want to coordinate their sessions

Reply:
This is now WIP, I am not yet sure how to represent that data. But I would think that proposing a Topic is routhly orthogonal to scheduling the topics. Maybe we could add a field like `temporalRestrictions` (which might not be rendered).